content/en/_index.md

```diff
@@ -62,4 +62,8 @@ Check out the [contribution guidelines](/docs/contrib/) to learn how to get invo
 
 <div class="text-center">We are a <a href="https://www.cncf.io">Cloud Native Computing Foundation</a> sandbox project.</div>
 
+<p>
+  <div class="text-center">The Linux Foundation® (TLF) has registered trademarks and uses trademarks. For a list of TLF trademarks, see <a href="https://www.linuxfoundation.org/trademark-usage">Trademark Usage</a></div>
```

Comment:
is it possible to make this text smaller than the previous line ("We are a Cloud..."), similar to what Dapr does: https://dapr.io/

Reply:
For some reason the surrounding block is in a `div` with the `h4` class applied. To make the text smaller, we would have to drop it from the styling. I assume this is embedded in the huge theme itself.

![image](https://github.com/user-attachments/assets/ddec947a-9b24-49fc-aea7-e2cb42307ec7)

Reply:
I pushed a change that fixes the issue!

Reply:
New layout:
![image](https://github.com/user-attachments/assets/3e5e770a-b467-48a4-9fe2-fb7c0bbcba0f)

Reply:
Thanks for the update @bacongobbler!  Are you happy with this latest version @kate-goldenring?